### PR TITLE
[WIP] Adds a DHCP server configuration guide

### PR DESCRIPTION
**Pull request by dougbtv**
Originally submitted as a stub document. Just using the contents of a guide that was provided to QE for validating this functionality.

The contents will need to be taken into a lab and made into a procedure that's reasonable for a user to both implement and maintain (it should be containerized, this uses the raw host itself).

Further description will be required.

### CODE REVIEW

File: modules/nw-dhcp-configuration-guide.adoc [hunk 1]
```diff
@@ -0,0 +1,42 @@
+[id="dhcp-configuration-guide_{context}"]
+= Creating a DHCP server
+
+A DHCP server can be created if there is not an existing DHCP server available to you to use for creating DHCP.
```
Comment (lamek) [inline modules/nw-dhcp-configuration-guide.adoc]: I would delete the last part of this sentence... "~~to you to use for creating DHCP~~."

File: modules/nw-dhcp-configuration-guide.adoc [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+
+.Prerequisites
+
+* [stub]
+
+.Procedure
+
+Doc for configure DHCP server (in container host)
+You need to configure 1 DHCP server in a node of cluster.
```
Comment (lamek) [inline modules/nw-dhcp-configuration-guide.adoc]: ..."on a node in the cluster."

**Review by lamek — COMMENTED** [review]
I know this is a first pass. Added some minor comments. The structure looks good so far.

**Comment by vikram-redhat** [comment]
@dougbtv would we still want to keep this PR open? Do you want someone from the team to work with you on this?

**Comment by jboxman** [comment]
@dougbtv, does Whereabouts supersede this?

**Comment by jboxman** [comment]
It's been 9 months; let's revisit if there is a need. Closing.

**Pull request closed without merging**